### ISSUE TITLE
improvement: BottomCupertinoTabItem

### ISSUE THREAD

**Issue by nissaba** [iJack93/bottom_cupertino_tabbar#17]
Hello. I am trying your BottomCupertinoTabbar and the class BottomCupertinoTabItem is only accepting svg files, and the class BottomCupertinoTabItem does not have parameter to give the coloring of the icons. 

I my test the icons do not get tinted with the colors given in the activeColor and inactiveColor params from the BottomCupertinoTabbar. They just got tinted white, and white is alse the color of the background. 

I think it would be beneficail to move away from using the string params pointing at a svg file and moving to acepting the Icon class instead. this wiil give users more capacity to fine tune the icons.

**Comment by nissaba** [comment]
It is possible also that the image of the svg are not showing up at all, in that case this ticket should become a bug.

**Comment by nissaba** [comment]
I re-coloured the fill color of my svg to use black inplace of grey that i was using this seams to have "
fixed" my color problem, I can now tint the svg. this is odd.

**Comment by iJack93** [comment]
Hello @nissaba, thank you for using my package.
That's an interesting suggestion, I will try to add it in the next release.

In the meantime, may you provide a sample test project with your use case with some sample tab item image in SVG format? 

The colors (for both the active/inactive states) are given from the `BottomCupertinoTabbar(...)` class, not from the single `BottomCupertinoTabItem`. If you run the example project in this repo, you can see that changing both `activeColor` and `inactiveColor` will change the color in the `BottomCupertinoTabItem`s respectively.
Thank you!

**Comment by iJack93** [comment]
@nissaba ?

**Comment by nissaba** [comment]
Sorry, I am on vacation at this time. will get back to you next week. cheers!

**Comment by nissaba** [comment]
In Material dart TabBar it is using tab that are defiened as 

```
const Tab({
    super.key,
    this.text,
    this.icon,
    this.iconMargin,
    this.height,
    this.child,
  })
```

where icon is defined as a widget `final Widget? icon;`
It could also jut be defined as an Icon as this would prevent some one setting the Icon as a TextField..

Usage:
```
Tab(
    icon: SvgPicture.asset(
        'assets/bike_icon.svg',
        color: Colors.white,
        width: 23,
        height: 23,
    ),
 )
```

I find that having this as an Icon give us a lot of flexibility tu use a SVG file or an icon from an Icon set.

**Comment by nissaba** [comment]
[samples.zip](https://github.com/user-attachments/files/17102369/samples.zip)

sample of the image I used. Original was a PDF vector image that I was using in my original iOS native app. I used Inkscape or word to extract the SVG (I forgot since then which one I used to get the SVG out of the pdf). this svg was not getting the tint originally, I had to edit the SVG to set it to black before it started working (using Inkscape).
